config.js

```diff
@@ -2,6 +2,6 @@
 
 (function () {
   require('./lib/main').config(
-    require('./lib/cli-options')(process.argv)
+    Object.assign({}, require('./lib/env-options'), require('./lib/cli-options')(process.argv))
```

Comment:
Let’s add some integration tests to verify behavior that documentation outlines

Reply:
added :)